src/occupation.jl

```diff
@@ -71,6 +71,35 @@ function compute_occupation(basis::PlaneWaveBasis{T}, eigenvalues;
                              Roots.Bisection(), atol=eps(T))
     end
 
+    if iszero(temperature)
```

Comment:
Maybe in general homogenise all temperature checks to `iszero` in this function.